monitoring/metrics/msmq-queue-length.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Setup Queue Length Metrics Reproting for the MSMQ Transport
+reviewed: 2018-05-11
+component: MetricsServiceControl
+related:
+  - samples/logging/metrics  
+---
+
+The component `NServiceBus.Metrics.ServiceControl.Msmq` monitors endpoints' queue length and passes that data to `NServiceBus.Metrics.ServiceControl` which in turn sends it to an instance of `ServiceControl.Monitoring` service.
```

Comment:
Is it worth creating a mermaid diagram here to show these relationships?

Reply:
added